opentelemetry-dotnet-contrib.sln

```diff
@@ -182,6 +182,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Instrumentati
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Instrumentation.Hangfire.Tests", "test\OpenTelemetry.Instrumentation.Hangfire.Tests\OpenTelemetry.Instrumentation.Hangfire.Tests.csproj", "{ED774FC3-C1C0-44CD-BA41-686C04BEB3E5}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Extensions.PersistentStorage.Api", "src\OpenTelemetry.Extensions.PersistentStorage.Api\OpenTelemetry.Extensions.PersistentStorage.Api.csproj", "{83ABFF05-952C-4301-9DCA-A033DBBA1C13}"

```

Comment:
I don't have an issue with this per-se, but FIY I think the current .NET style is to call these type of projects "abstractions" eg: `OpenTelemetry.Extensions.PersistentStorage.Abstractions`